cms/plugin_pool.py

```diff
@@ -37,10 +37,67 @@ def clear(self):
         self.plugins = {}
         self.patched = False
 
+    def validate_templates(self, plugin=None):
+        """
+        Plugins templates are validated at this stage
+
+        """
+        if plugin:
+            plugins = [plugin]
+        else:
+            plugins = self.plugins.values()
+        for plugin in plugins:
+            if (plugin.render_plugin and not type(plugin.render_plugin) == property
```

Comment:
`type() is not property`? (or just `is` -- not sure about the order of operations for the `not` here)